assets/stylesheets/kitten/components/modals/modal.scss

```diff
@@ -0,0 +1,84 @@
+/// Modal
+///
+/// @group modals
+///
+/// @example scss - usage
+///
+///   @include k-Modal((
+///     z-index: 10
+///   ));
+///
+/// @example html
+///
+///   <div class="k-Modal k-Modal--afterOpen">
+///     <div class="k-Modal__paragraph"> … </div>
+///     <div class="k-Modal__action"> … </div>
+///   </div>
+
+@mixin k-Modal($options) {
+  .k-Modal {
+    position: relative;
+    max-height: calc(100vh - #{k-px-to-rem(16px)} * 2);
```

Comment:
Vu qu'on le répète 2 fois, on pourrait sortir `#{k-px-to-rem(16px)} * 2` dans une variable.

Que représente le `16px` ?